apps/base-docs/docs/tools/oracles.md

```diff
@@ -119,14 +119,39 @@ See [this guide](https://docs.ora.io/doc/oao-onchain-ai-oracle/develop-guide/tut
 
 ## Pyth
 
-[Pyth](http://pyth.network/) offers 250+ [price feeds](https://pyth.network/price-feeds) for Base.
+The [Pyth Network](https://pyth.network/) is the largest first-party Oracle network, delivering real-time data across [a vast number of chains](https://docs.pyth.network/price-feeds/contract-addresses). Pyth introduces an innovative low-latency [pull oracle design](https://docs.pyth.network/documentation/pythnet-price-feeds/on-demand), where users can pull price updates on-chain when needed, enabling everyone in the blockchain environment to access that data point most efficiently. Pyth network updates the prices every **400ms**, making Pyth the fastest on-chain oracle.
```

Comment:
s/the largest/one of the largest/
s/blockchain/onchain/
s/on-chain/onchain/
s/the fastest on-chain/
s/one of the fastest onchain/

Reply:
@aditya520 I still don't see the commit that made these changes, could you please update?

Reply:
Updated.